openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -183,28 +210,40 @@ def test_jwt_access_token(self, client_attr):
         data = json.loads(response.content)
         self.assertIn('expires_in', data)
         self.assertEqual(data['token_type'], 'JWT')
-        self.assert_valid_jwt_access_token(data['access_token'], self.user, data['scope'].split(' '))
+        self.assert_valid_jwt_access_token(
+            data['access_token'],
+            self.user,
+            data['scope'].split(' '),
+            should_be_restricted=False,
+        )
 
-    def test_restricted_jwt_access_token(self):
+    @ddt.data(
+        (False, True, settings.DEFAULT_JWT_ISSUER),
+        (True, False, settings.RESTRICTED_APPLICATION_JWT_ISSUER),
```

Comment:
Missing test case?
`(True, True, settings.DEFAULT_JWT_ISSUER),`
Would `RESTRICTED_APPLICATION_JWT_ISSUER` always be used when `enforce_jwt_scopes_enabled` is set to `True`?